components/pages/profile/payments/controller.tsx

```diff
@@ -28,27 +30,41 @@ export default function PaymentsPage({
   const isNetworkPayments = !!router?.query?.networkName && !!payments?.length;
   const fiatSymbol = state?.Settings?.currency?.defaultFiat?.toUpperCase();
 
+  const {
+    data: prices,
+    isLoading,
+    isSuccess,
+  } = useCoingeckoPrice(payments.flatMap(({ payments }) => payments.map(payment => ({
+    address: payment?.issue?.transactionalToken.address,
+    chainId: payment?.issue?.transactionalToken.chain_id
+  })).reduce((acc, value) => acc.concat(value), [])));
```

Comment:
I think this `reduce` is not necessary here, `flatMap + map` already returns an array of objects `{ address, chainId }`